src/app/app.component.html

```diff
@@ -11,13 +11,24 @@
               <div *ngFor="let item of cart">
                 <div *ngIf="item.ingredient_name">
                   {{item.product_name}} med {{item.ingredient_name}} ({{item.price}} kr) x{{item.item_quantity}}
+                  <button class="btn-remove" id="remove_{{item.product_name}}-{{item.ingredient_name}}"
+                          (click)="removeProduct(item.product_id, item.product_name, item.price, item.ingredient_id, item.ingredient_name)">
```

Comment:
Why not pass the `item` only then as an argument then extract all those fields within the remove product method? 🤔